Makefile

```diff
@@ -103,19 +103,17 @@ ready: fmt-amend ## Amend current commit with fmt changes
 
 ##@ Documentation
 
-ANTORA_DOCS=./antora/docs
-ANNOTATIONS_JSON=$(ANTORA_DOCS)/rule_annotations.json
-
-$(ANNOTATIONS_JSON):
-# Use jq sort_by here because otherwise the order is different every time
-	@opa inspect --annotations --format json $(POLICY_DIR) | jq -S '.annotations |= sort_by(.location.file, .location.row)' > $@
-
-.PHONY: clean-annotations
-clean-annotations:
-	@rm -f $(ANNOTATIONS_JSON)
+# These two should produce identical output
```

Comment:
Can we say "annotations-js and annotations-opa" instead of "These two" ?

Reply:
Sure.